docs/methoddocs/filleeventqueries.md

```diff
@@ -1 +1,300 @@
 # File Event Queries
+
+```eval_rst
+.. autoclass:: py42.clients.file_event.FileEventClient
+    :members:
+    :show-inheritance:
+```
+
+```eval_rst
+.. autoclass:: py42.sdk.queries.fileevents.file_event_query.FileEventQuery
+    :members:
+    :show-inheritance:
+```
+
+# Filter Classes
+
+Each of the following classes can be used to filter Forensic Search results based on the class's corresponding event
+attribute. Call the appropriate classmethod on your desired filter class with the `value` you want to match, and it
+will return a FilterGroup object that can be passed to `FileEventQuery`'s `all()` or `any()` methods to combine complex
+queries that match multiple filter rules.
+
+Example:
+
+To search for events concerning a certain set of documents, you can use the `FileName` and `MD5` filter classes to
+construct FilterGroups that will search for matching filenames or (in case someone renamed the sensitive file) the
+known MD5 hashes of the files:
+
+    filename_filter = FileName.is_in(['confidential_plans.docx', 'confidential_plan_projections.xlsx'])
+    md5_filter = MD5.is_in(['133765f4fff5e3038b9352a4d14e1532', 'ea16f0cbfc76f6eba292871f8a8c794b'])
```

Comment:
We should link to the searches userguide!

Reply:
Good call!